src/styles/styles.scss

```
@@ -27,3 +55,67 @@
     width: 150px;
   }
 }
+
+.input-container {
+  max-width: 20rem;
+  margin: 0.5rem auto 2rem auto;
```

Similar comment here, the margin can be write as 
```suggestion
  margin: 0.5rem auto 2rem;
```
since the left-right are both `auto`